src/data/http/http.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { refreshToken as refreshTokenService } from '../auth/authServices';
+
+const axiosInstance = axios.create({
```

Comment:
I think `http` is a good name for this file. And It seems that http is more suitable name then axiosInstance.
I suggest to rename:
```suggestion
const http = axios.create({
```